pkgs/top-level/all-packages.nix

```diff
@@ -11467,6 +11467,10 @@ with pkgs;
 
   openfortivpn = callPackage ../tools/networking/openfortivpn { };
 
+  openobserve = darwin.apple_sdk_11_0.callPackage ../servers/monitoring/openobserve {
+    apple_sdk = darwin.apple_sdk_11_0;
+  };
```

Comment:
Does this work?

```suggestion
  openobserve = callPackage ../servers/monitoring/openobserve {
    stdenv = if stdenv.isDarwin then overrideSDK stdenv "11.0" else stdenv;
  };
```

Reply:
Not entirely. It looks like there's also a separate `rustPlatform` attribute provided by the `darwin.apple_sdk_11_0.callPackage`.